README.md

```diff
@@ -69,7 +69,7 @@ This image required few file to be mounted in the container's `/data` folder.
 ```
 .aria2/session.txt  (empty file)
 .aria2/aria2.log    (empty file)
-.aria2/aria2.conf   (aria2 configuration file, not webui-aria2 conf)
+.aria2/aria2.conf   (aria2 configuration file, not webui-aria2 conf) must contains at least `enable-rpc=true` and `rpc-listen-all=true`

```

Comment:
There is no way you don't want to use it.
But, i'm not such a fan of splitted configuration. We'll have some rpc conf harcoded in the Dockrfile and `rpc-secret` or even `rpc-listen-port` in the aria config file. It does not make real sense to me to have such a splitted config, and most users won't bother with `rpc-secret` if it does work out of the box without a config file.

And it will make `aria2.conf` having 2 lines that will have no effect and may lead to confusion.